odooconnector_base/models/product_supplierinfo.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd rather have this model and purchase in a separate module odooconnector_purchase (same for sales etc.) and I would like to follow Odoo common structure
In general some users will have different needs:
* simple product or partner (use this one)
* PO2SO (use odooconnector_base+odooconnector_purchase+odooconnector_sale)
* SO2SO (use odooconnector_base+odooconnector_sale)
* etc.
depending on the final needs we can assemble the basic models accordingly